### PR TITLE
Fix closing tag of opt out iframe

### LINKED ISSUE #12665: iframe opt-in/opt-out code is not properly closed
Hi, after updating to 3.4.0 and checking HTML opt-in/opt-out code, the generated code looks like this:

`<iframe
        style="border: 0; height: 200px; width: 600px;"
        src="https: // ... /index.php?module=CoreAdminHome&action=optOut&language=en&backgroundColor=&fontColor=&fontSize=&fontFamily="
        >`

However, this breaks my page completely, everything below the embedded code above ends-up hidden inside the iframe. This is because the code itself for iframe is not properly closed:

`<iframe
        style="border: 0; height: 200px; width: 600px;"
        src="https: // ... /index.php?module=CoreAdminHome&action=optOut&language=en&backgroundColor=&fontColor=&fontSize=&fontFamily="
        ></iframe>`

Not sure if this is because I updated Matomo, maybe it does not happen in fresh installations. However, check if you missed the closing `</iframe>` tag in your HTML template.

Thanks!

### PR DESCRIPTION
**Pull request by sgiehl**
fixes #12665

### CODE REVIEW

File: plugins/PrivacyManager/angularjs/opt-out-customizer/opt-out-customizer.directive.html [hunk 1]
```diff
@@ -44,9 +44,9 @@ <h3>HTML code to embed on your website</h3>
     <pre piwik-select-on-focus>&lt;iframe
         style="border: 0; height: 200px; width: 600px;"
         src="{{ optOutCustomizer.iframeUrl }}"
-        ></iframe></pre>
+        &gt;&lt;/iframe&gt;</pre>
     <p ng-bind-html="'CoreAdminHome_OptOutExplanationIntro'|translate:'&lt;a  href=\'' + optOutCustomizer.iframeUrl + '\' rel=\'noreferrer\' target=\'_blank\'>':'&lt/a>'">
```
Comment (fdellwing) [inline plugins/PrivacyManager/angularjs/opt-out-customizer/opt-out-customizer.directive.html]: I think there is a `;` missing at the end of the entity.

Comment (sgiehl) [reply plugins/PrivacyManager/angularjs/opt-out-customizer/opt-out-customizer.directive.html]: right, didn't notice

**Review by diosmosis — APPROVED** [review]
LGTM

**Comment by diosmosis** [comment]
One screenshot test failure: https://builds-artifacts.matomo.org/matomo-org/matomo/fixoptoutiframe/26702/UIIntegrationTest_admin_privacy_settings.png

**Comment by sgiehl** [comment]
updated the ui file :)

**Comment by tsteur** [comment]
It looks like it is still missing the closing iframe in the example markup itself?

**Comment by tsteur** [comment]
all good my fault... it works :)

**Pull request merged**